configs/cardano/mainnet/download.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+set -euo pipefail
+
+curl https://book.play.dev.cardano.org/environments/mainnet/config.json \
+    > config.json
+curl https://book.play.dev.cardano.org/environments/mainnet/db-sync-config.json \
+    > db-sync-config.json
+curl https://book.play.dev.cardano.org/environments/mainnet/genesis.json \
+    > genesis.json
```

Comment:
```suggestion
curl https://book.play.dev.cardano.org/environments/mainnet/genesis.json \
    > genesis.json
```

That file does not seem to exist? 🤔